docs/guides/extension_points.rst

```diff
@@ -63,6 +63,9 @@ If you want to provide learners with new content experiences within courses, opt
    * - **External Graders**
      - Hold, Stable
      - An external grader is a service that receives learner responses to a problem, processes those responses, and returns feedback and a problem grade to the edX platform. You build and deploy an external grader separately from the edX platform. An external grader is particularly useful for software programming courses where learners are asked to submit complex code. See the `external grader documentation`_ for details.
+   * - **TinyMCE Plugins**
```

Comment:
People might not know what TinyMCE is. This should say "TinyMCE (Visual Text/HTML Editor) Plugins"

Reply:
Yeah, that's a great remark. I added it in the guide I added too 🙂 

This has been addressed in [`e6803fa - Updates the extension points and adds a guide on how to install custom tinymce plugins`](https://github.com/edx/edx-platform/pull/25695/commits/e6803fab65f0e35badf21addb214c46819b5fc91) 👍🏼